### ISSUE TITLE
Look into loading other file types for 3d objects

### ISSUE THREAD

**Issue by uberlinuxguy** [OpenParsec/openparsec#87]
@Cheeseness mentioned GLTF
@slime73 mentioned OBJ
@CrazySpence mentioned vrml2.

We should take a look at the 3 of 'em and see what looks the easiest/has a good C/C++ lib, and just roll with it. 

Requirements:
- [ ] has a library interface to help with loading, so we don't have to manually parse the file.
- [ ] has compatibility with windows, linux, and mac (all current versions)
- [ ] can be easily translated to a GenObject struct
- [ ] look at the animation stuff for GenObjects?
- [ ] ???

**Comment by uberlinuxguy** [comment]
GLTF:
- [ ]A possible library exists: https://github.com/syoyo/tinygltf
- [ ]Library has compatability for mac + clang, linux + GCC/CLANG, windows VS2015 and later. 
- [ ]GenObject?
- [ ]Animation?

**Comment by slime73** [comment]
GLTF is the most fully featured of those formats, I think it makes sense to aim for it as long it can support everything the game needs.

I noticed genobject/odt have some "interesting" / 90's era material settings per face, figuring that stuff out - and how much of it is actually used - is probably one part of moving to one of these new formats.

**Comment by uberlinuxguy** [comment]
Yeah I was just looking for libraries for the other two, VRML2 seems to be a very old format, might be hard to find a library for it.   OBJ which was a wavefront thing(?) Seems pretty old too, the couple I found on github referencing that were unmaintained.

**Comment by CrazySpence** [comment]
I like libraries.

the online conversion sites also seem to support this so i can turn a bunch of wing commander ships i've been stock piling for years into it with the drop of a hat </secret plan>
